package.json

```diff
@@ -64,6 +64,7 @@
         "test": "run-s test:node test:browser",
         "test:node": "env HEDERA_SDK_TEST='' run-s test:unit:node test:integration:node",
         "test:unit:node": "env HEDERA_SDK_TEST='' nyc mocha -r @babel/register -r chai/register-expect 'test/unit/*.js'",
+        "test:unit:single": "env HEDERA_SDK_TEST='' nyc mocha -r @babel/register -r chai/register-expect",
```

Comment:
Not entirely sure why this was added to be honest. If you'd like to test a single unit test you should be able to use
```
pnpm test:unit:node -- -g "Mix signing and serialization.*"
```

Reply:
Thanks wasn't aware of that. Will remove.